Classes/Systems/ShortcutHandler.swift

```diff
@@ -57,7 +57,7 @@ struct ShortcutHandler {
         // Bookmarks
         let bookmarkIcon = UIApplicationShortcutIcon(templateImageName: "bookmarks")
         let bookmarkItem = UIApplicationShortcutItem(type: Items.bookmarks.rawValue,
```

Comment:
Should I update these variable names to match the plural form too?
That is, `bookmarksIcon` and `bookmarksItem`.

Reply:
Don't think that's needed 👍